packages/intrepid2/unit-test/Shared/ArrayTools/test_01.hpp

```diff
@@ -91,11 +91,11 @@ namespace Intrepid2 {
       Teuchos::oblackholestream oldFormatState;
       oldFormatState.copyfmt(std::cout);
 
-      typedef typename
-        Kokkos::Impl::is_space<DeviceSpaceType>::host_mirror_space::execution_space HostSpaceType ;
+      using DeviceExecSpaceType = typename DeviceType::execution_space;
+      using HostExecSpaceType = Kokkos::DefaultExecutionSpace;
```

Comment:
@kyungjoo-kim, isn't the device the default exec space for a Cuda build?  It seems confusing to define a typedef that seems to say that this is the HostExecSpaceType...

Reply:
The default execution space has uvm memory as its default memory space. To test the code without the uvm, we need ``Kokkos::Device<Kokkos::Cuda,Kokkos::CudaSpace>`` and execution space is derived fromthe device type.

Reply:
Ah.... you are right. It should be DefaultHostExecutionSpace. I misunderstand your comments.

Reply:
Looking a bit more closely at this, you don't actually use the HostExecSpaceType anywhere except to print the configuration.  I would suggest getting rid of this typedef, since it suggests (to me anyway) that we'll be using it somewhere in the code below.  I would also change the text in the configuration output to say simply "DefaultExecutionSpace" instead of saying anything about the host.  Presumably the configuration output will include the fact that UVM is or is not enabled.